# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cyberjack/ctapi-header.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ctapi-cyberjack-2.0.8/ctapi/Makefile.orig	2005-04-19 14:57:02.000000000 +0200
+++ ctapi-cyberjack-2.0.8/ctapi/Makefile	2005-04-19 14:57:48.000000000 +0200
@@ -15,8 +15,8 @@
 install: all
 	install -o root -g root -d $(INSTALL_PREFIX)/lib
 	install -o root -g root libctapi-cyberjack.so $(INSTALL_PREFIX)/lib
-	install -o root -g root -d $(INSTALL_PREFIX)/include
-	install -o root -g root include/ctapi.h $(INSTALL_PREFIX)/include
+	install -o root -g root -d $(INSTALL_PREFIX)/include/cyberjack
+	install -o root -g root include/ctapi.h $(INSTALL_PREFIX)/include/cyberjack
 
 %.o: %.c
 	$(CC) $(CFLAGS) -o $@ -c $^
